### DISCUSSION TITLE
Can you update the online docs please?

### DISCUSSION THREAD

**Discussion by systemcrash** [lldpd/lldpd#638] [General]
Looks like the last run failed: https://github.com/lldpd/lldpd/actions/runs/7520182668

**Comment by systemcrash** [comment]
Also nice if you could trigger auto-build per push :)

**Comment by vincentbernat** [comment]
It is up-to-date, this is the right action: https://github.com/lldpd/lldpd.github.io/actions/runs/7520181461

**Comment by systemcrash** [comment]
Ah yes, sorry, different repo 🤦

**Comment by systemcrash** [comment]
Can it be updated with the latest edits from my last PR?

**Reply by vincentbernat** [reply]
It's updated when there is a new release. The manual pages on the site match the current release.

**Reply by systemcrash** [reply]
Yeah, for release content consistency, it should not be updated.

**Comment by systemcrash** [comment]
I see. My last changes were only language, no functionality changes were
documented. So I think they could go in. Was just confused when I was
reading the man page thinking "sure I updated that bit..."
